assume/common/outputs.py

```diff
@@ -357,7 +357,7 @@ async def on_stop(self):
         for query in queries:
             try:
                 df = pd.read_sql(query, self.db)
-            except (OperationalError, ProgrammingError):
+            except Exception as e:
```

Comment:
We don't want to catch every exception without notice.
You might want to add a second except block with different hadling..?

Reply:
yeah true